rust/gitxetcore/src/git_integration/repo_migration/migration.rs

```diff
@@ -780,17 +775,14 @@ pub async fn migrate_repo(
                 match obj.kind().unwrap_or(ObjectType::Any) {
                     ObjectType::Tree => {
                         let deps = {
-                            if in_note_conversion_stage
-                                && commit_tree_oids.contains(&oid)
-                                && !known_non_note_oids.contains(&oid)
-                            {
+                            if in_note_conversion_stage && is_note_oid {
                                 get_note_tree_dependents(&src, obj, &full_tr_map)
```

Comment:
Actually, tree OIDs in the tree field of a note tree is itself a note tree.   Just the OIDs in the path field are not.